components/txn_types/Cargo.toml

```diff
@@ -31,3 +32,7 @@ tikv_alloc = { path = "../tikv_alloc" }
 tikv_util = { path = "../tikv_util", default-features = false }
 collections = { path = "../collections" }
 log_wrappers = { path = "../log_wrappers" }
+
+[dev-dependencies]
+bitflags = "1.0"
+rand = "0.7"
```

Comment:
Why are they necessary?

Reply:
`bitflags` is a mistake, `rand` is used for test